packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1684,6 +1811,21 @@ export class Combobox
       <div ariaHidden="true" class={CSS.floatingUIContainer} ref={setFloatingEl}>
         <div class={classes} ref={setContainerEl}>
           <ul class={{ list: true, "list--hide": !open }}>
+            {this.selectAllEnabled &&
+              this.selectionMode !== "single" &&
+              this.selectionMode !== "single-persist" && (
+                <calcite-combobox-item
+                  ariaLabel={this.messages.selectAll}
+                  class={CSS.selectAll}
+                  id={`${this.guid}-select-all-enabled`}
+                  ref={this.setSelectAllComboboxItemReferenceEl}
+                  role="option"
+                  selected={this.updateSelectAllState()}
```

Comment:
This function sounds like a function that returns void but is returning boolean here. Can we rename this to better describe what it is doing?

Reply:
Should this just be set to `this.selectAllComboboxItemReferenceEl.selected`?